BP/common/culture/cultures/00_byzantine.txt

```diff
@@ -1333,7 +1340,6 @@
 			Yberia Ytilia
 		}
 
-		dynasty_of_location_suffix = "dynns_ites" # So "ites"
```

Comment:
You said you would comment out these as opposed to removing.

Reply:
Fixed.